Strings/More.jsonc

```diff
@@ -114,15 +106,15 @@
 
 	"不具合報告等は現在 Twitter 上で受け付けています。": {
 		"ja": "不具合報告等は現在 Twitter 上で受け付けています。",
-		"en": "Thanks for your bug reports via Twitter.",
+		"en": "If you'd like to report a bug, contact us on Twitter.",
 		"zh-Hans": "請在 Twitter 上提交错误报告。",
 		"zh-Hant": "請在 Twitter 上提交錯誤報告。",
 		"zh-HK": "請在 Twitter 上提交錯誤報告。"
 	},
 
 	"履歴をすべて削除": {
 		"ja": "履歴をすべて削除",
-		"en": "Delete all history",
+		"en": "Clear all history",
```

Comment:
It's more common to say Clear to remove contents (空になって) because Delete implies the removal of the entire thing.

From Google Chrome:
![image](https://user-images.githubusercontent.com/6291467/95545602-102a5700-0a4a-11eb-8460-612e4000eb43.png)

Reply:
Clear ↓
```js
var array = [1, 2, 3, 4];
array = []; // clear
```

Delete ↓
```js
var array = [1, 2, 3, 4];
array = null; // delete
```

Reply:
OK! I understood that clearly. Thank you!